packages/core/content-manager/admin/src/features/DocumentContext.tsx

```diff
@@ -54,7 +60,11 @@ const DocumentContextProvider = ({
    * one of the root level document's relations.
    */
   const [currentDocumentMeta, changeDocument] = React.useState<DocumentMeta>(initialDocument);
-  const document = useDocument(currentDocumentMeta);
+  const params = React.useMemo(
```

Comment:
I don't know if the useMemo is necessary or not

Reply:
If you remember we discussed regarding the removal of useMemo and useCallback if not strictly necessary so feel free to remove it